init-mqtt.h

```diff
@@ -109,9 +109,21 @@ void initMqtt() {
   
   loadMqttConfig();
   //temporary mqtt configs. for testing...
-  setMqttConfig("m23.cloudmqtt.com", 12925,"tlwhlgqr", "g-VQc5c6w7eN");
+  //setMqttConfig("m23.cloudmqtt.com", 12925,"tlwhlgqr", "g-VQc5c6w7eN");
+  setMqttConfig("m13.cloudmqtt.com", 12730,"wnezdvgh", "ycm1xiuzX936");
   if (isMqttConfigSet) {
     mqttClient.setServer(mqttConfig.server, mqttConfig.port);    
   }
 }
+
+void publishInt(const char* topic, int d) {
```

Comment:
Will we publish only ints?

Reply:
Don't know at the moment
We'll introduce another methods in case we need

Reply:
yes - exactly